app/models/user.rb

```diff
@@ -1,3 +1,4 @@
 class User < ApplicationRecord
-  
-end
+  validates :name, presence: true
+  has_many  :reviews
```

Comment:
to really complete this relationship, we want to have another line here that says `has_many :books, through: :reviews`...that would allow us to call `book.users` without having to iterate through each review